DQM/Integration/python/clients/hlt_dqm_clientPB-live_cfg.py

```diff
@@ -42,9 +42,9 @@
 process.load('HLTrigger.Timer.fastTimerServiceClient_cfi')
 process.fastTimerServiceClient.dqmPath = "HLT/TimerService"
 # timing VS lumi
-process.fastTimerServiceClient.doPlotsVsScalLumi  = True
+process.fastTimerServiceClient.doPlotsVsOnlineLumi = True
 process.fastTimerServiceClient.doPlotsVsPixelLumi = False
-process.fastTimerServiceClient.scalLumiME = cms.PSet(
+process.fastTimerServiceClient.onlineLumiME = cms.PSet(
     folder = cms.string('HLT/LumiMonitoring'),
```

Comment:
This is the only place I could find where the parameter names appeared, and needed to be changed.